cocktails/easy/wikipedia_pina_colada.md

```diff
@@ -0,0 +1,15 @@
+# Textbook (Wikipedia inspired) Piña Colada
+
+## Ingredients
+
+* 1 part white rum
+* 1 part (Alpro) coconut drink
```

Comment:
not coconut cream?

Reply:
coconut drink is intended!

Reply:
(in my experience coconut cream is harder to come across, but am not sure if it would make the recipe better)